### DISCUSSION TITLE
No bookmarklets working after using this for first time

### DISCUSSION THREAD

**Discussion by SkyAtSchool** [3kh0/ext-remover#49] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
The bookmarklets worked well, but after I turned off my Chromebook, every bookmarklet I have isn't working.
Does anyone know why this happened or/and how to fix it?

**Comment by CheesyPoofs3** [comment]
I have the same problem

**Reply by GinyuSpecialForce** [reply]
when you turn your Chromebook off, it updates, and it got patched so it won't work anymore unless they figure another way to bypass the patch

**Reply by SkyAtSchool** [reply]
How come it happened right after? Did all the update do was remove all bookmarklets?
I did find out about this:
![image](https://user-images.githubusercontent.com/85582285/195710752-66941e11-4afb-44b3-aa61-f65e5031be70.png)
I think my school somehow detected I disabled GoGuardian & light speed filter and knew I was using bookmarklets to do it, so they disabled it for me only.

**Comment by 3kh0** [comment, marked as answer]
@SkyAtSchool You said you have the javascript base listed in the URLBlocklist, we are still trying to find a solution in #38, so please post anything there!

**Comment by Synaptic149** [comment]
ChromeOS 106 patched it
